README.md

```diff
@@ -1,22 +1,140 @@
 # Pimcore Elasticsearch Plugin
 
-## What it does
+The Elasticsearch plugin for Pimcore Saves/updates contents (text only) of document's (editables) and assets to Elasticsearch. It also provides a query builder to search the index.
 
-* Saves/updates contents (text only) of Document's editables to Elasticsearch
- 
-## Configure
+## Features
 
-Copy the distribution config file (`elasticsearchplugin.xml.dist`) in the root of the plugin folder 
+* Automatic indexing of documents and assets to elasticsearch when created/updated in Pimcore admin. 
+* Hooks to add custom properties to the index.
+* A simple query builder to retrieve indexed documents. Supports querying, filtering, sorting and pagination.
+
+## Installation via composer
+
+The recommended method to install the Pimcore Elasticsearch PLuing is via Composer.
+
+1 - Add byng/pimcore-elasticsearch-plugin as a dependency in your project's composer.json file (change version to suit your version of Elasticsearch):
+```json
+    {
+        "require": {
+            "byng/pimcore-elasticsearch-plugin": "~3.0"
+        }
+    }
+```
+
+2 - Download and install Composer:
```

Comment:
I don't think this is relevant, to us, and even if it were, I wouldn't put the actual steps here, because they could become invalid at any point. It'd just reword step 3 (which would become step 2), so it was like:

3 - Install or update your dependencies via [Composer](https://getcomposer.org/):
```bash
composer install
// or
composer update byng/pimcore-elasticsearch-plugin
```